db/migrate/20160318213755_create_gem_download.rb

```diff
@@ -0,0 +1,10 @@
+class CreateGemDownload < ActiveRecord::Migration
+  def change
+    create_table :gem_downloads do |t|
+      t.integer :rubygem_id, null: false
+      t.integer :version_id, null: false
```

Comment:
> I am not a big fan of polymorphic assocs

I hear you. :smile: 

At Kickstarter, we have a `metrics` table with a unique string `name` and an integer `value`. We could use that here too. We've gotten a lot of mileage out of such a generic table. We don't use it for everything, but it's an easy place to start for 80% of business metrics we want to easily query.

The names could be obvious things like `total-gem-downloads`; `rake-total-gem-downloads`, `rake-v11.1.1-downloads`.